docs/getting-started/self-hosting.md

```diff
@@ -121,6 +125,53 @@ docker-compose up -d
  docker-compose logs dashboard
 ```
 
+### Advanced: Running netbird behind an existing reverse-proxy
+
+If you want to run netbird behind your own reverse-proxy, some additional configuration-steps have to be taken to [Step 2](#step-2--prepare-configuration-files).
+
+:::info
+Not all reverse-proxies are supported as netbird uses *gRPC* for various components.
+:::
+
+#### Configuration for netbird
+
+In `setup.env`:
+- Set ```NETBIRD_DOMAIN``` to your domain, e.g.  `demo.netbird.io`
+- Set ```NETBIRD_DISABLE_LETSENCRYPT=true```
+
+In `base.env`:
+- Set ```NETBIRD_MGMT_API_PORT``` to your reverse-proxy TLS-port (default: 443)
+- Set ```NETBIRD_SIGNAL_PORT``` to your reverse-proxy TLS-port
+
+If required, change/remove the port-mappings for `dasbhoard`, `signal` and `management`-services in `docker-compose.yml.tmpl`.
+
+:::tip info
+The `coturn`-service still needs to be directly accessible under your set-domain as it uses UDP for communication.  
```

Comment:
Same comment about using setup.env

Reply:
Not sure how to properly mention this.  
e.g. with traefik no port-mapping is necessary, so removing them is best.

We could remove them dynamically if they are unset using e.g. `sed`, but a proper templating-action would be better. (requiring if-blocks)  

As we have multiple templates (currently the default and traefik), an input-argument for `configure.sh` would also be an option.  
e.g. `./configure.sh docker-compose.yml.tmpl.traefik`

Reply:
you can keep the mentions for all variables, but instead of driving users to edit `base.setup.env` drive them to use `setup.env` as the PR allows for that

Reply:
Ah, i thought you meant the `docker-compose.yml.tmpl` in general.

Still not 100% happy with the amount of edits one has to do, but we can fix that with some changes to the configure later.